torchdp/compute_dp_sgd_privacy.py

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env python3
```

Comment:
For consistency with Pytorch-DP variable names, let us use `alpha` and `alphas` instead of `order` and `orders` :). Also, let us use `sample_rate` instead of `q`  and `noise_multiplier` instead of `sigma` in this Python file :).

Reply:
Done in two separate commits : first with internal variables modified only, second where bash arguments are adapted to be consistent, in case the latter is not suitable.